src/components/layout.css

```diff
@@ -528,3 +522,31 @@ main {
   top: 2000px;
   width: 3px;
 }
+
+/* TOC */
+details,
+.toc {
+  cursor: pointer;
+}
+.toc h6 {
+  display: inline-block;
+  margin: 0;
+}
+.toc a {
+  font-family: var(--sans-serif);
+  text-decoration: none;
+  border-bottom: 1px dotted var(--gray3);
+}
+
+.toc li p {
+  margin: 0;
+}
+
+/* 3rd Party plugin styles. */
+/* Styles for gatsby-remark-autolink-headers plugin*/
+
+.autolink-headers {
+  position: absolute;
+  left: -4px;
+  fill: var(--orange5);
```

Comment:
nit: `orange5` is too dark... makes the link icon look 'dirty'. Let's please stick to the brighter `orange3` ([it was originally the color for the `#` heading prefixes](https://github.com/nodejs/nodejs.dev/pull/186/files#diff-e68d1c0fd6fbd176f45675bb2ac2119dL429)).
```suggestion
  fill: var(--orange3);
```

Reply:
`orange3` is too light. I can't see it. Bad for accessibility. We don't want that.

![](https://on.ahmda.ws/b5dfc4/c)

It passes with either `--orange5` or `--green6`.